app1/admin.py

```diff
@@ -1,3 +1,19 @@
-from django.contrib import admin
+#from django.contrib import admin
 
 # Register your models here.
+from django.contrib import admin
+from .models import YourModel  # Import your model(s) here
+
+@admin.register(YourModel)
+class YourModelAdmin(admin.ModelAdmin):
+    list_display = ('field1', 'field2', 'field3')  # Add fields you want to display in the admin list view
+    search_fields = ('field1', 'field2')  # Add fields you want to search in the admin
+
+# If you have more models, you can register them in a similar fashion
+# Example:
+# from .models import AnotherModel
+# 
+# @admin.register(AnotherModel)
+# class AnotherModelAdmin(admin.ModelAdmin):
+#     list_display = ('fieldA', 'fieldB')
+#     search_fields = ('fieldA', 'fieldB')
```

Comment:
**Suggestion**: Consider creating a separate module or file for admin model registrations. This approach improves maintainability and scalability, especially as more models are added to the project. <br> **Code Suggestion**: 
 ```
 + # In admin.py or a new module, e.g., admin_register.py
 + from django.contrib import admin
 + from .models import YourModel  # Import your model(s) here
 + ...
 + class YourModelAdmin(admin.ModelAdmin):
 ```